hack/ccm/Makefile

```diff
@@ -0,0 +1,46 @@
+REGISTRY=quay.io/kabhat
+IMG=powervs-cloud-controller-manager
+VPC_CONTROLLER_COMMIT?=""
+POWERVS_CLOUD_CONTROLLER_COMMIT?=""
```

Comment:
It will be good if we document these commits somewhere, can be even in the same makefile too. Whenever we want to bump the version - we can just update the makefile and build the image for the change.